pkg/kn/commands/service/service.go

```diff
@@ -33,8 +33,9 @@ const (
 
 func NewServiceCommand(p *commands.KnParams) *cobra.Command {
 	serviceCmd := &cobra.Command{
-		Use:   "service",
-		Short: "Manage Knative services",
+		Use:     "service",
+		Short:   "Manage Knative services",
```

Comment:
```suggestion
		Short:   "Manage Knative services (aliases: ksvc, services)",
```

Reply:
as done for leaf commands

Reply:
So if there is one alias only should it be `(alias:` ?

Reply:
yes

Reply:
Done